Makefile

```diff
@@ -69,10 +75,11 @@ test-postgresql:
 
 #
 # Generate targets to build Go commands.
+# Note: ratelimit-tester is in hockeypuck/ratelimit/cmd/, others are in hockeypuck/server/cmd/
```

Comment:
Wondering if this is a good idea, maybe all commands should be in the same place? Not going to die on this hill though... ;-)